.github/actions/common-cd-setup/action.yaml

```diff
@@ -73,7 +73,7 @@ runs:
       run: |
         sed -i 's|<IMAGE>|'${{ env.image }}'|' $GITHUB_WORKSPACE/deployment/${{ inputs.app_input }}/deployment.yaml
         kubectl apply -f $GITHUB_WORKSPACE/deployment/${{ inputs.app_input }}/deployment.yaml
-        [[ "${{ inputs.app_input }}" == "deploy" ]] && kubectl apply -f $GITHUB_WORKSPACE/deployment/${{ inputs.app_input }}/all.yaml
+        [[ "${{ inputs.app_input }}" == "deploy" ]] && kubectl apply -f $GITHUB_WORKSPACE/deployment/${{ inputs.app_input }}/all.yaml || true
```

Comment:
@iv1310 It's better to put it on feature related step.

Reply:
https://github.com/dappforce/polkaverse/blob/ffa477597db2cecba8dfaea1c4fb0f9fa0edd43f/.github/actions/common-cd-setup/action.yaml#L57C1-L65C1

Reply:
Okay will do